version.json

```
@@ -1,3 +1,3 @@
 {
-  "version": "v1.9.2"
+  "version": "v1.9.3"
```

```suggestion
  "version": "v1.10.0"
```

for the Go version bump I'd go minor

---

Done